lib/rubocop/cop/rspec/return_from_stub.rb

```diff
@@ -49,7 +49,7 @@ class ReturnFromStub < Cop
 
         def_node_matcher :and_return_value, <<-PATTERN
             (send
-              (send nil :receive (...)) :and_return $(...)
+              (send nil :receive $(...)) :and_return $(...)
```

Comment:
It would be better if I could match only the and_return() part, this way offense won't mark `receive(:msg)` part and the code would be cleaner. If anyone have suggestion how to do so, please share